fastlane/Fastfile

```diff
@@ -97,6 +99,11 @@ lane :upgrade_swift do |options|
     puts "\e[31m\e[5mDon't forget to update the swift version in the dependecies: .podspecs & cartfile'\e[0m"
 end
 
+lane :deploy_api_reference do
+	sh("../tools/make-doc.sh")
+	sh("cd ../build/doc && git add . && git commit -m 'update api reference' && git push")
```

Comment:
couldn't use the git commit command of fastlane because I need to be in the folder that checkouts gh-pages and I could only do it this way